app/src/main/java/io/islnd/android/islnd/app/fragments/FeedFragment.java

```diff
@@ -103,8 +139,59 @@ public void onPause() {
         getContext().unregisterReceiver(mStopRefreshReceiver);
     }
 
-    public void startNewPostActivity() {
+    private void startNewPostActivity() {
         Intent newPostIntent = new Intent(mContext, NewPostActivity.class);
         startActivityForResult(newPostIntent, NEW_POST_RESULT);
     }
+
+    private void setNotificationBadgeCount() {
+        if (mNotificationCount > 0 && mNotificationCount < 100) {
+            mNotificationBadgeCount.setVisibility(View.VISIBLE);
+            mNotificationBadgeCount.setText(Integer.toString(mNotificationCount));
+        } else if (mNotificationCount >= 100) {
+            mNotificationBadgeCount.setText(getString(R.string.notification_badge_99_plus));
```

Comment:
Does this `else if` also need to set the badge count to be visible? I looks like if we jump from 0 notifications to 100+, the badge will stay invisible.